include/effects.h

```diff
@@ -674,8 +686,31 @@ typedef struct DebuffFXData {
 } DebuffFXData; // size = 0x40
 
 typedef struct GreenImpactFXData {
-    /* 0x00 */ char todo[0];
-} GreenImpactFXData; // size = unknown
+    /* 0x00 */ u16 unk_00;
+    /* 0x02 */ char unk_02[2]; // likely padding
```

Comment:
Both `unk_02` and `unk_56` can be omitted without changing the compiled code. Is it preferable to leave them here just in case?

Reply:
We typically leave them in for now.